WordPress/twentytwentyfive
# Services Team Photos

Fixes: https://github.com/WordPress/twentytwentyfive/issues/79

Created the team photo pattern. We're adding translatable strings later to avoid issues with DQA.

![services team photos](https://github.com/user-attachments/assets/671b7383-7890-4b13-8248-d08af003ecaa)

### Testing Instructions
- Create a page.
- Add the pattern.
- Confirm that it looks like the designs (spacing/colors/etc.).
